src/main/java/seedu/address/model/customer/Customer.java

```diff
@@ -139,4 +133,64 @@ public String toString() {
         return builder.toString();
     }
 
+    /**
+     * Copy of customer with new commissions.
```

Comment:
I think this JavaDocs might not adhere to the style guide, maybe something like `Copies...` would be better?